source/Outfit.cpp

```diff
@@ -118,11 +118,25 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
-	if(node.Size() >= 2)
+	if(node.Size() == 2)
```

Comment:
I would think that outfit variants would behave similarly to ship variants. That is, outfit variants should display with the same name as their parent. You'd then save the variant name to a separate string that is used when a ship saves its outfits or a mission saves its require and outfit nodes, just as ship variants do when a mission saves its give ship nodes.